brave/brave-core
# Storybook: Created connected-with-site-panel

Resolves <https://github.com/brave/brave-browser/issues/14878>

Created connected-with-site-panel and added to storybook

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. `npm run storybook`
2. Confirm that the storybook shows new component.
3. Check functionality of the new component.


https://user-images.githubusercontent.com/40611140/112348302-50defb00-8c8d-11eb-87d1-c9f7f2ecff90.mov
